src/Cake.Svn/Add/SvnAddSettings.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Cake.Svn.Add
+{
+    /// <summary>
+    /// Settings for <see cref="SvnAdder"/>.
+    /// </summary>
+    public sealed class SvnAddSettings : SvnSettings
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SvnAddSettings"/> class.
+        /// </summary>
+        public SvnAddSettings()
+        {
+            Depth = SvnDepth.Infinity;
+            AddParents = true;
+        }
+
+        /// <summary>
+        /// If TRUE, recurse up path's directory and look for a versioned directory. If found,
+        /// add all intermediate paths between it and path. If not found, fail with 
+        /// SVN_ERR_CLIENT_NO_VERSIONED_PARENTS.
+        /// </summary>
+        public bool AddParents { get; set; }
+
+        /// <summary>
+        /// If force is not set and path is already under version control, return the error
+        /// SVN_ERR_ENTRY_EXISTS. If force is set, do not error on already-versioned items.
+        /// When used on a directory in conjunction with the recursive flag, this has the
+        /// effect of scheduling for addition unversioned files and directories scattered
+        /// deep within a versioned tree.
+        /// </summary>
+        public bool Force { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether auto properties should be enabled on the file or folder.
+        /// </summary>
+        public bool NoAutoProps { get; set; }
```

Comment:
IMHO `AutoProperties` would be more intuitive to use than `NoAutoProps`, even though it is different to the SharpSvn property, but since we shield SharpSvn API anyway from the user of the addin this shouldn't be an issue here. 

If we switch to `AutoProperties` just make sure to also switch the default value.